### PR TITLE
fix: stores minors

### PR DESCRIPTION
**Pull request by manuel-calavera**

### CODE REVIEW

File: src/v2/stores/accounts/detail.js [hunk 1]
```diff
@@ -45,4 +45,4 @@ decorate(Store, {
 
 const AccountDetailStore = new Store();
 
-export default observable(AccountDetailStore);
```
Comment (manuel-calavera) [inline src/v2/stores/accounts/detail.js]: observable can't be used here. observable values can be JS primitives, references, plain objects, class instances, arrays and maps.

File: src/v2/stores/accounts/timeline.js [hunk 2]
```diff
@@ -50,7 +50,6 @@ class Store {
 decorate(Store, {
   init: action.bound,
   start: observable,
-  accountTimeline: observable,
```
Comment (manuel-calavera) [inline src/v2/stores/accounts/timeline.js]: i think it is not used anywhere for now.

File: src/v2/stores/accounts/detail.js (hunk 1 above)
Comment (sunnygleason) [reply src/v2/stores/accounts/detail.js]: this makes sense - we should apply it to all the newer stores. If you can do it, great! If not,
I can take a pass through & take care of it...

**Pull request merged**